apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'The Supabase AI Hackathon'
+description: Build an Open Source Project over 10 days. 5 prize categories.
+author: tyler_shukert
+image: lw7-hackathon/thumbnail.png
+thumb: lw7-hackathon/thumbnail.png
```

Comment:
```suggestion
thumb: lw7-hackathon/thumbnail.png
launchweek: 7
```

Reply:
Can you please just add this `launchweek: 7` to the meta data?
I'll use it to show some additional stuff at the end of the post.

Reply:
Yup, sounds great!